doc/spec_exceptionHandling.dox

```diff
@@ -30,6 +30,8 @@ namespace ChimeraTK {
 
 \subsection spec_exceptionHandling_intro_comments (*) Comments
 
+- \anchor exceptionHandling_comment_a_5_1 \ref exceptionHandling_a_5_1 "5.1" Uncorrelated recovery groups are merged if a Device is added which uses backends from both.
```

Comment:
Ah you have that here. I would make a full point out of it, this will also help for testing (as it requires a dedicated test case).